cocotb_build_libs.py

```diff
@@ -311,9 +311,12 @@ def _get_common_lib_ext(include_dir, share_lib_dir):
     ]
     if os.name == "nt":
         libcocotb_sources += ["libcocotb.rc"]
+    libcocotb_defines = [("COCOTB_EMBED_EXPORTS", "")] + _extra_defines
+    if os.name != "nt":
+        libcocotb_defines += [("PYTHON_SO_LIB", find_libpython.find_libpython())]
     libcocotb = Extension(
         os.path.join("cocotb", "libs", "libcocotb"),
-        define_macros=[("COCOTB_EMBED_EXPORTS", ""), ("PYTHON_SO_LIB", find_libpython.find_libpython())] + _extra_defines,
+        define_macros=libcocotb_defines,
```

Comment:
This change is probably unnecessary. Unused decls are ignored.

Reply:
The path on windows contains `\`, distutils does not properly escape them and therefore compilation would fail. So this is just a convenient workaround. https://travis-ci.org/github/cocotb/cocotb/jobs/719676704#L599